### ISSUE TITLE
Switching to hex/dec/bin output hangs GUI

### ISSUE THREAD

**Issue by danbim** [wisebed/wisegui#40]
Sometimes it is not even possible to close the browser tab (Chrome)

**Comment by master-lincoln** [comment]
Quasi duplicate of #39...Should be way better now after #41 (at least on modern machines)

**Comment by danbim** [comment]
Not really a duplicate! Even if performance is made better there still may be the issue (e.g. on not-so-modern machines or machines under load) that the GUI hangs. We need a workaround here that keeps the GUI responsive!
